website/docs/concepts/gps_service/README.md

```diff
@@ -53,6 +59,10 @@ It shows the difference in position between a "moving" receiver and a "fixed" re
 Check the protobuf definition for more details:
 [gps_pb2.RelativePositionFrame](https://github.com/farm-ng/farm-ng-amiga/blob/main/protos/farm_ng/gps/gps.proto#L73-L97)
 
+- `/ecef`:ECEF message.
+Check the protobuf definition for more details:
+[gps_pb2.EcefCoordinates](https://github.com/farm-ng/farm-ng-amiga/blob/main/protos/farm_ng/gps/gps.proto#L109-L117)
```

Comment:
In a perfect world, we should only use line numbers if it is pinned to a tag. Otherwise it is subject to change. But I'd be hesitant to pin to tags because we may update the proto and that increases risk of someone seeing an old version of the proto when they want to see the newest.

Anyways, this is fine (and clearly what we did before) but it's not ideal. The best case would be to generate an API for the protos as part of the doc build process and link to that

Reply:
I agree with you here. The best thing moving forward is to generate API for the protos as part of the doc build process.
Alternatively, we could have a single link that takes users to gps.proto. But then, it wouldn't make much sense to have three copies of the same link (one for each message). I will resolve the comment for now, but I am happy to brainstorm together ideas for optimizing our documentation.